docs/release-notes/rockcraft-1-7-0.rst

```diff
@@ -0,0 +1,101 @@
+.. _release-1.7.0:
+
+Rockcraft 1.7.0 release notes
```

Comment:
I also removed some sections, like the "Requirements" one, which would be empty for this particular release.

Reply:
It was decided that the _Requirements_ section should be mandatory for each page, so it will have to stay in. It's in effect boilerplate.

Reply:
Alright, but in this case we should review the content - the default template lists "Python 3.x or higher", but in the case of the applications the "official" means of distribution is the snap. In this case, there's no requirement on the user even having Python, but they do need a snap-capable system. Shall I draft a list of requirements here that we can possibly use for the other applications too?

Reply:
Yes, please! Feel free to crib from Snapcraft's system reqs, too: https://canonical-snapcraft.readthedocs-hosted.com/en/latest/reference/system-requirements/

Reply:
done! Thanks for the cheatsheet